MixItUp.Base/Services/External/StreamlabsService.cs

```diff
@@ -16,35 +16,23 @@ namespace MixItUp.Base.Services.External
 {
     public class StreamlabsDonation
     {
-        [JsonProperty("donation_id")]
-        public int ID { get; set; }
+        [JsonProperty("_id")]
+        public string ID { get; set; }
 
         [JsonProperty("name")]
         public string UserName { get; set; }
+
         [JsonProperty("message")]
         public string Message { get; set; }
 
-        [JsonProperty("amount")]
+        [JsonProperty("formatted_amount")]
         public string AmountString { get; set; }
 
-        [JsonProperty("created_at")]
-        public long CreatedAt { get; set; }
-
-        [JsonIgnore]
-        public double Amount
-        {
-            get
-            {
-                if (this.AmountString.ParseCurrency(out double result))
-                {
-                    return result;
-                }
-                return 0;
-            }
-        }
+        [JsonProperty("amount")]
+        public double Amount { get; set; }
 
         [JsonIgnore]
-        public DateTimeOffset CreatedAtDateTime { get { return StreamingClient.Base.Util.DateTimeOffsetExtensions.FromUTCUnixTimeSeconds(this.CreatedAt); } }
+        public DateTimeOffset CreatedAtDateTime { get { return DateTimeOffset.UtcNow; } }
```

Comment:
The websocket packet doesn't have any indication of when it was created, not sure if this is used by us or not, but... not sure if there is a better option.

Reply:
In theory since the socket is "real time", the Create Date should just be the current time. It was used previously to exclude old donations since we had to do polling.